Documentation/network/concepts/ipam/index.rst

```diff
@@ -26,6 +26,10 @@ Multiple CIDRs per node      ❌                     ❌                       
 Dynamic CIDR/IP allocation   ❌                     ❌                       ✅          ✅          ✅              ✅                ❌
 ============================ ====================== ======================== =========== =========== =============== ================= ===============
 
+Changing the IPAM mode of an existing cluster is impossible.
```

Comment:
Maybe..?

```suggestion
Changing the IPAM mode of an existing cluster is strongly discouraged as it may cause persistent connectivity disruption for existing workloads.
```

Reply:
This plus the "you must reinstall Cilium ..." should make it clear both why and also how to make the change if necessary.

Reply:
(I prefer my proposal at the bottom of this feedback, but I'm also open to this proposal. In particular I think that the previous text that states "it is impossible" followed by "here's how to do it" is confusing.)